server/controllers/entities/lib/has_works_labels_occurrence.coffee

```diff
@@ -69,4 +67,13 @@ hasBnfOccurrence = (authorEntity, worksLabels)->
   # is wrong and we can't know which
   if bnfIds?.length isnt 1 then return false
   getBnfAuthorWorksTitles bnfIds[0]
-  .then (titles)-> _.haveAMatch titles, worksLabels
+  .then createOccurencDocs worksLabelsPattern, authorEntity
+
+createOccurenceDocs = (worksLabelsPattern, authorEntity)->
```

Comment:
in the rest of the code, `docs` is associated with full CouchDB documents: what about `createOccurenceObj`?

Reply:
or `createOccurences`